components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/editor/js/operator-finder/operator-finder.js

```diff
@@ -320,27 +359,51 @@ define(['jquery', 'lodash', 'log','remarkable', 'handlebar', 'designViewUtils',
                 });
             }
             var keyResult = [], descriptionResult = [], combineResults;
-            this._operators.forEach(function (e, i) {
+            this._operators.forEach(function (operatorExtension, i) {
                 var result = {
-                    fqn: hasToken(e.fqn, tokens),
-                    description: hasToken(e.description, tokens)
+                    fqn: hasToken(operatorExtension.fqn, tokens),
+                    description: hasToken(operatorExtension.description, tokens)
                 };
                 if (result.fqn.status) {
-                    keyResult.push({
-                        fqn: e.fqn,
-                        htmlFqn: result.fqn.text,
-                        type: e.type,
-                        description: result.description.text,
-                        index: i
-                    });
+                    if (isNotInstalledExtension(operatorExtension)) {
+                        keyResult.push({
+                            extension: getNotInstalledExtensionObject(operatorExtension),
+                            status: "Not-Installed",
```

Comment:
use constants please

Reply:
88cbc98 this commit will fix this change request